### PR TITLE
PBKDF2

### PR DESCRIPTION
**Pull request by SebastianBoe**
See commit messages

### CODE REVIEW

**Review by vili-nordic — APPROVED** [review]

File: doc/nrf/libraries/nrf_security/doc/driver_config.rst [hunk 1]
```diff
@@ -87,33 +87,46 @@ Key Derivation Function
...
-| TLS 1.2 EC J-PAKE to PMS | :kconfig:option:`CONFIG_PSA_WANT_ALG_TLS12_ECJPAKE_TO_PMS` |
-+--------------------------+------------------------------------------------------------+
++--------------------------+---------------------------------------------------------------+
+| KDF algorithm            | Configuration option                                          |
++==========================+===============================================================+
+| HKDF                     | :kconfig:option:`CONFIG_PSA_WANT_ALG_HKDF`                    |
++--------------------------+---------------------------------------------------------------+
+| PBKDF2_HMAC              | :kconfig:option:`CONFIG_PSA_WANT_ALG_PBKDF2_HMAC`             |
++--------------------------+---------------------------------------------------------------+
+| PBKDF2_AES_CMAC_PRF_128  | :kconfig:option:`CONFIG_PSA_WANT_ALG_PBKDF2_AES_CMAC_PRF_128` |
```
Comment (joerchan) [inline doc/nrf/libraries/nrf_security/doc/driver_config.rst:97]: I think it is more natural to put: `PBKDF2-AES-CMAC-PRF-128`. This is how the PSA documentation describes it in text.

File: doc/nrf/libraries/nrf_security/doc/driver_config.rst [hunk 2]
```diff
@@ -87,33 +87,46 @@ Key Derivation Function
...
-| TLS 1.2 PSK to MS        | :kconfig:option:`CONFIG_PSA_WANT_ALG_TLS12_PSK_TO_MS`      |
-+--------------------------+------------------------------------------------------------+
-| TLS 1.2 EC J-PAKE to PMS | :kconfig:option:`CONFIG_PSA_WANT_ALG_TLS12_ECJPAKE_TO_PMS` |
-+--------------------------+------------------------------------------------------------+
++--------------------------+---------------------------------------------------------------+
+| KDF algorithm            | Configuration option                                          |
++==========================+===============================================================+
+| HKDF                     | :kconfig:option:`CONFIG_PSA_WANT_ALG_HKDF`                    |
++--------------------------+---------------------------------------------------------------+
+| PBKDF2_HMAC              | :kconfig:option:`CONFIG_PSA_WANT_ALG_PBKDF2_HMAC`             |
```
Comment (joerchan) [inline doc/nrf/libraries/nrf_security/doc/driver_config.rst:95]: I think it is more natural to put `PBKDF2-HMAC`. This is how the PSA documentation describes it in text.

**Review by joerchan — COMMENTED** [review]
Suggestion on different naming in text vs code.

File: doc/nrf/libraries/nrf_security/doc/driver_config.rst (hunk 2 above)
Comment (SebastianBoe) [reply doc/nrf/libraries/nrf_security/doc/driver_config.rst]: fixed

File: doc/nrf/libraries/nrf_security/doc/driver_config.rst (hunk 1 above)
Comment (SebastianBoe) [reply doc/nrf/libraries/nrf_security/doc/driver_config.rst]: fixed

**Review by joerchan — APPROVED** [review]

**Comment by SebastianBoe** [comment]
updated SHA

**Pull request merged**